appveyor.yml

```diff
@@ -57,7 +57,7 @@ install:
   # compiled extensions and are not provided as pre-built wheel packages,
   # pip will build them from source using the MSVC compiler matching the
   # target Python version and architecture
-  - "%CMD_IN_ENV% pip install \"pytest>=4.3.0\" pythonnet six cefpython3==66.0"
+  - "%CMD_IN_ENV% pip install \"pytest>=4.3.0\" .[cef]"
```

Comment:
What does this syntax `.[cef]`do?

Reply:
Installs the local directory as a package with the extra `cef`.

Reply:
aka, DRYs your dependency lists.